pygments/lexers/chapel.py

```diff
@@ -133,4 +134,7 @@ class ChapelLexer(RegexLexer):
             (words(known_types, suffix=r'\b'), Keyword.Type),
             (r'[^()]*', Name.Other, '#pop'),
         ],
+        'attributename': [
+            (r'([a-zA-Z_][.\w$]*)', Name.Decorator, '#pop'),
```

Comment:
Secondly, is `Name.Attribute` more appropriate here? Additionally, the parenthesis here can also be omitted. 

```suggestion
            (r'[a-zA-Z_][.\w$]*', Name.Attribute, '#pop'),
```

Reply:
I was not certain on the best choice for token type here. Our "attributes" don't really align with any of the existing types as I read them in https://pygments.org/docs/tokens/. They look and function something like Java annotations, which is why I thought maybe `Decorator` was appropriate, but I would defer to someone with more knowledge of the token system in `pygments`.

Reply:
`Decorator` sounds good then, thanks!